frontend/src/App.js

```diff
@@ -15,6 +15,9 @@ import SchoolIndexPage from "main/pages/SchoolIndexPage";
 
 import CoursesCreatePage from "main/pages/CoursesCreatePage";
 import CourseIndexPage from "main/pages/CourseIndexPage";
+//import CoursesShowPage from "main/pages/CoursesShowPage";
```

Comment:
Commented out code does not belong in a PR that's been approved and code reviewed by a team member.
Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed